src/main/java/com/nbcampif/ifstagram/global/config/WebSecurityConfig.java

```diff
@@ -37,6 +38,7 @@ protected SecurityFilterChain configure(HttpSecurity http) throws Exception {
             .permitAll() // resources 접근 허용 설정
             .requestMatchers("/", "/api/v1/auth/**", "/oauth2/**")
             .permitAll() // 인증 관련
+            .requestMatchers("/api/v1/admin/**").hasRole(UserRole.ADMIN.toString())
```

Comment:
```suggestion
            .requestMatchers("/api/v1/admin/**").hasRole(UserRole.ADMIN.name())
```
name이 더 맞는 것 같은데 toString으로 사용하신 이유가 있으신가요?

Reply:
name을 몰랐네요. 수정하겠습니다.